### PR TITLE
equationsToMatrix

### PR DESCRIPTION
**Pull request by latot**
Fixes https://github.com/cbm755/octsympy/issues/576

Hi, well again a new function, this still i need add the doc and tests, add info in the pytave section, etc, so don't check it for the moment plis.

Edited, I found the problem, fixing.

Thx. Cya.

### CODE REVIEW

File: inst/@sym/equationsToMatrix.m [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+function [A b] = equationsToMatrix(varargin)
+
+  s = symvar([varargin{:}]);
+
+  cmd = {'eqns = list(_ins); vars = list()'
+         'if not _ins[-1].is_Matrix:'
+         '    for i in reversed(xrange(len(_ins))):'
+         '        if _ins[i].is_Symbol:'
+         '            vars.append(_ins[i])'
+         '            del _ins[-1]'
```
Comment (latot) [inline inst/@sym/equationsToMatrix.m]: This line don't works.

File: inst/@sym/equationsToMatrix.m [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+         '        else:'
+         '            break'
+         'else:'
+         '    for i in _ins[-1]:'
+         '        if i.is_Symbol:'
+         '            vars.append(i)'
+         '    del _ins[-1]'
```
Comment (latot) [inline inst/@sym/equationsToMatrix.m]: Same.

**Comment by latot** [comment]
Okidokis, Now works!

**Comment by latot** [comment]
Code ready and formatted!

File: inst/@sym/equationsToMatrix.m [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+
+  if ~s
+    error('Cannot convert to matrix form because the system does not seem to be linear.');
+  end
+
+end
+
+
+%%Matlab Tests
```
Comment (cbm755) [inline inst/@sym/equationsToMatrix.m]: We need to come up with our own tests.  Or we could take from a text book (which we cite).

Comment (latot) [inline inst/@sym/equationsToMatrix.m]: we can't use directly the matlab tests?

Comment (cbm755) [inline inst/@sym/equationsToMatrix.m]: Maybe its a grey area, but I don't think its a good idea.

And just a reminder: reading their docs is ok, but we should *never* look inside their m-files.

Comment (latot) [inline inst/@sym/equationsToMatrix.m]: xD i'm agree, that its a very bad idea.

but don't is impossible some codes be similars... (its a hard point)
(i don't have matlab)

**Comment by latot** [comment]
ready!
why i need search in books? octave do the work!
```
a=[];
for i=1:3
  for j=1:4
    a(i, j) = (-1)^fix(rand*10)*fix(rand*10);
  end
end
```

**Comment by cbm755** [comment]
Ok, I reviewed this and sent a PR to you.  Further changes could probably be done post-merge.

Note to self: rebase this before merging for clean history (because of examples noted above).

**Comment by cbm755** [comment]
Merged as #684.

**Pull request closed without merging**